docs/source/shaders/material/as_glass.rst

```diff
@@ -168,7 +168,7 @@ Screenshots
    :width: 10%
    :title:
 
-   Sharp glass with long specular highlight tails, creating the appearance of a thin veil over the specular highlights. Using the Student t distribution with a specular roughness of 0.05, specular spread of 0.25, and with a IOR set to 1.64.
+   Sharp glass with long specular highlight tails, creating the appearance of a thin veil over the specular highlights. Using the Student t-distribution with a specular roughness of 0.05, specular spread of 0.25, and with a IOR set to 1.64.
```

Comment:
*Student's* (also below) and we're good to merge ;)

Reply:
I think it was added in the previous commit, but let me recheck. Btw, thanks for the historical reference on the distribution. I think i might need to fix the references to the Student t-distribution on our Standard Surface material as well though.

Reply:
yep, there was a reference there as well, fixed below.